react/features/base/participants/actions.js

```diff
@@ -503,6 +503,35 @@ export function participantMutedUs(participant, track) {
     };
 }
 
+/**
+ * Action to create a fake screen share participant.
+ *
+ * @param {(JitsiLocalTrack|JitsiRemoteTrack)} track - JitsiTrack instance.
+ * @returns {Function}
+ */
+export function createFakeScreenShareParticipant(track) {
```

Comment:
I've moved the fake SS creation logic out of the tracks middleware to allow it to be dispatched in other places